pkg/generate/app/cmd/newapp_test.go

```diff
@@ -66,8 +66,8 @@ func TestAddArguments(t *testing.T) {
 			unknown:    []string{},
 		},
 		"source": {
-			args:    []string{".", testDir, "git://server/repo.git"},
-			repos:   util.StringList([]string{".", testDir, "git://server/repo.git"}),
+			args:    []string{".", testDir, "git://github.com/openshift/origin.git"},
+			repos:   util.StringList([]string{".", testDir, "git://github.com/openshift/origin.git"}),
```

Comment:
if this test is going to end up cloning this repo, we should pick a smaller one.  origin is huge.